autogpt/memory/base.py

```diff
@@ -4,6 +4,20 @@
 from autogpt.singleton import AbstractSingleton
 
 
+def save_vec_num(vec_num):
+    with open("vec_num.txt", "w") as file:
+        file.write(str(vec_num))
+
+
+def load_vec_num():
+    try:
+        with open("vec_num.txt", "r") as file:
+            vec_num = int(file.read())
+            return vec_num
+    except FileNotFoundError:
+        return 0
+
+
```

Comment:
These should be workspace dependent on their contexts cc: @collijk

Reply:
@ntindle I removed the save/load_vec_num functions in my latest commit. Now it accesses the vector count directly after initializing pinecone